components/contact.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import styles from './contact.module.css';
+import utilsStyles from '../styles/utils.module.css';
+
+export default function Contact() {
+	const [values, setValues] = useState({
+		name: '',
+		email: '',
+		message: '',
+	});
+
+	const handleOnChange = (e) => {
+		const { name, value } = e.target;
+		setValues({ ...values, [name]: value });
+	};
+
+	const handleSubmitBtn = (e) => {
+		e.preventDefault();
+	};
+
+	return (
+		<section className={utilsStyles.bg_white} id="contact">
+			<div className={styles.contact_section_container}>
+				<h2>
+					<span className={utilsStyles.font_accent}>C</span>ONTACT
+				</h2>
+				<p className={styles.thank_you}>
```

Comment:
This is responsive, but the font might look better if bigger. I suggest using a global h4 or h5 instead of a modified paragraph tag because there are other places this could be useful.

Reply:
We can adjust this test size later 👍  Thanks for checking!